src/app/app.module.ts

```diff
@@ -113,6 +113,10 @@ import { RavenExceptionHandler } from './shared/exception.handler';
 import { ForgeWizardModule } from './space/forge-wizard/forge-wizard.module';
 
 
+// ngrx stuff
+import { EffectsModule } from '@ngrx/effects';
```

Comment:
The imports are in 3 sections. (all the files should be following this layout)
Angular, external, then internal. Each section is alphabetical. Please move this to about line 11.